dashboard/src/components/Cards/IssuesList.tsx

```diff
@@ -132,9 +132,7 @@ const IssuesList = ({
           tagPills?.unshift(
             <Tooltip>
               <TooltipTrigger className="cursor-default">
-                <Badge variant="greenTag">
-                  {formatMessage({ id: 'global.new' })}
-                </Badge>
+                <div className="starburst bg-red aspect-square w-[20px]"></div>
```

Comment:
nit to make it even more visible
```suggestion
                <div className="starburst bg-red aspect-square w-[24px]"></div>
```